scripts/3rdparty.py

```diff
@@ -554,6 +554,31 @@ def makeenv(envlist):
     execute('make -j', logfile)
     install_files('src/sox.h', os.path.join(builddir, 'include'))
     install_files('src/.libs/libsox.a', os.path.join(builddir, 'lib'))
+
+elif name == 'libunwind':
+	download(
+		'http://download.savannah.nongnu.org/releases/libunwind/libunwind-%s.tar.gz' % ver,
+		'libunwind-%s.tar.gz' % ver,
+		logfile,
+		vendordir)
+	extract('libunwind-%s.tar.gz' % ver,
+			'src/libunwind-%s' % ver)
+	os.chdir('src/libunwind-%s' % ver)
+	execute('./configure --host=%s %s %s %s' % (
+        toolchain,
+        makeenv(envlist),
+        makeflags(workdir, toolchain, deplist, variant=variant),
```

Comment:
On my system, I also need to add `cflags='-fPIC'` here and `'--disable-minidebuginfo'` bellow (but it was not necessary on alpine for some reason).